contrib/README.md

```
@@ -0,0 +1,47 @@
+Supporting assets that help you wire Shiplog into a larger deployment workflow.
+
+## Hooks
+
+- `hooks/pre-receive.shiplog` – sample server-side guard that enforces (requires: git ≥2.35, jq ≥1.6, bash):
+  - fast-forward pushes to `refs/_shiplog/journal/*` and `refs/_shiplog/anchors/*`
+  - commit signatures (GPG or SSH allowed signers)
+  - author allowlists pulled from the active policy
+
+To install on a bare repository:
+
+#!/bin/bash
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)